assets/js/translator.js

```diff
@@ -447,12 +447,35 @@ function getPairs() /*: JQueryPromise<any> */ {
             }
         });
 
-        for(var srcLang in pairs) {
-            // Default for new users is first available pair
-            curSrcLang = srcLang;
-            curDstLang = pairs[srcLang][0];
-            break;
+        // Default for new users is first available Browser Preference Language pair
```

Comment:
Could you move all of this logic into a separate function?

Reply:
Done. :)